composer.json

```diff
@@ -1,21 +1,33 @@
 {
 	"name": "silktide/prospect-php-client",
 	"description": "PHP client for integrating Prospect into your projects.",
+
 	"require": {
-		"php": "^7.4"
+		"php": "^7.4",
+		"ext-json": "*",
+		"guzzlehttp/guzzle": "~6.0"
 	},
+
 	"require-dev": {
 		"phpunit/phpunit": "^8.5"
-	},
+    },
+
 	"authors": [
 		{
 			"name": "Greg Bowler",
 			"email": "greg.bowler@g105b.com"
```

Comment:
This was auto-generated from my git credentials. Should I change my git/github credentials to use my Silktide email?

Reply:
I don't think it really matters in terms of git credentials, as long as we can identify that it's you, it doesn't really matter.

It's purely from the perspective that if someone is looking for the author of this open source library to get help from, that should probably be your work email so that should you move on, that would get redirected appropriately 👍